create.py

```diff
@@ -97,6 +97,14 @@ def create_submission(author, path, language):
         cargo.write(f"\n[[bin]]\nname = \"{submission_name}\"\npath = \"{submission_file}\"\n")
         print("[+] added submission to Cargo.toml")
 
+    # Create a symlink to workspace if it is a Golang project
+    if language == "go":
+        workspace_directory = "workspace" + path[1:]
```

Comment:
Using `os.path.join()` would be cleaner and more readable than `+ path[1:]`. Same with `os.path.realpath()` or `os.path.abspath()` to get the absolute path